charts/akeyless-secure-remote-access/Chart.yaml

```diff
@@ -14,7 +14,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 
-version: 0.21.2
+version: 0.22.0
```

Comment:
You should rebase at first:
**This branch is out-of-date with the base branch**
and check - might be version already changed

Reply:
Rebased.
Version was 0.21.1, now 0.21.2, new will be 0.22.0